GateWatcher/aioniq_ecs/_meta/fields.yml

```
@@ -0,0 +1,1299 @@
+  name: gatewatcher.ioc.targeted_organizations
+  type: text
+
+gatewatcher.ioc.targeted_platforms:
+  description: ioc targeted_platforms field
+  name: gatewatcher.ioc.targeted_platforms
+  type: text
+
+gatewatcher.ioc.targeted_sectors:
+  description: ioc targeted_sectors field
+  name: gatewatcher.ioc.targeted_sectors
+  type: text
+
+gatewatcher.ioc.threat_actor:
+  description: ioc threat_actor field
+  name: gatewatcher.ioc.threat_actor
+  type: text
+
+gatewatcher.ioc.tlp:
+  description: ioc tlp field
+  name: gatewatcher.ioc.tlp
+  type: text
+
+gatewatcher.ioc.type:
+  description: ioc type field
+  name: gatewatcher.ioc.type
+  type: text
+
+gatewatcher.ioc.updated_date:
+  description: ioc updated_date field
+  name: gatewatcher.ioc.updated_date
+  type: datetime
+
+gatewatcher.ioc.usage_mode:
+  description: ioc usage_mode field
+  name: gatewatcher.ioc.usage_mode
+  type: text
+
+gatewatcher.ioc.value:
+  description: ioc value field
+  name: gatewatcher.ioc.value
+  type: text
+
+gatewatcher.krb5.cname:
+  description: krb5 cname field
+  name: gatewatcher.krb5.cname
+  type: text
+
+gatewatcher.krb5.encryption:
+  description: krb5 encryption field
+  name: gatewatcher.krb5.encryption
+  type: text
+
+gatewatcher.krb5.error_code:
+  description: krb5 error_code field
+  name: gatewatcher.krb5.error_code
+  type: text
+
+gatewatcher.krb5.failed_request:
+  description: krb5 failed_request field
+  name: gatewatcher.krb5.failed_request
+  type: text
+
+gatewatcher.krb5.msg_type:
+  description: krb5 msg_type field
+  name: gatewatcher.krb5.msg_type
+  type: text
+
+gatewatcher.krb5.realm:
+  description: krb5 realm field
+  name: gatewatcher.krb5.realm
+  type: text
+
+gatewatcher.krb5.sname:
+  description: krb5 sname field
+  name: gatewatcher.krb5.sname
+  type: text
+
+gatewatcher.krb5.weak_encryption:
+  description: krb5 weak_encryption field
+  name: gatewatcher.krb5.weak_encryption
+  type: boolean
+
+gatewatcher.malcore.analyzed_clean:
+  description: malcore analyzed_clean field
+  name: gatewatcher.malcore.analyzed_clean
+  type: number
+
+gatewatcher.malcore.analyzed_error:
+  description: malcore analyzed_error field
+  name: gatewatcher.malcore.analyzed_error
+  type: number
+
+gatewatcher.malcore.analyzed_infected:
+  description: malcore analyzed_infected field
+  name: gatewatcher.malcore.analyzed_infected
+  type: number
+
+gatewatcher.malcore.analyzed_other:
+  description: malcore analyzed_other field
+  name: gatewatcher.malcore.analyzed_other
+  type: number
+
+gatewatcher.malcore.analyzed_suspicious:
+  description: malcore analyzed_suspicious field
+  name: gatewatcher.malcore.analyzed_suspicious
+  type: number
+
+gatewatcher.malcore.analyzers_up:
+  description: malcore analyzers_up field
+  name: gatewatcher.malcore.analyzers_up
+  type: number
+
+gatewatcher.malcore.code:
+  description: malcore code field
+  name: gatewatcher.malcore.code
+  type: number
```

In the legacy format, this field is a `keyword` (see [fields.yml](https://github.com/SEKOIA-IO/intake-formats/blob/main/GateWatcher/aioniq/_meta/fields.yml#L21-L24))
The change of type will lead to a collision of type in the indexation engine.

```suggestion
gatewatcher.malcore.code:
  description: malcore code field
  name: gatewatcher.malcore.code
  type: keyword
```

---

Proposal fix in PR #1276